components/checkbox/index.css

```diff
@@ -533,7 +533,7 @@ governing permissions and limitations under the License.
 
   .spectrum-Checkbox-checkmark,
   .spectrum-Checkbox-partialCheckmark {
-    color: var(--highcontrast-checkbox-background-color-default, var(--mode-checkbox-checkmark-color, var(--spectrum-checkbox-checkmark-color)));
+    color: var(--highcontrast-checkbox-background-color-default, var(--mod-checkbox-checkmark-color, var(--spectrum-checkbox-checkmark-color)));
```

Comment:
@pfulton are we ok not having the incorrect spelling retained as a fallback in this case?

Reply:
Yeah, I think that's fine. After we merge and release this, I can let the bug reporter know about this change.